tools/osx-pkg-preinstall.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# TODO Can this be done inside the .pmdoc?
+# TODO Can we extract $PREFIX from the installer?
+cd /usr/local/bin
+rm -rf ../lib/node_modules/npm
```

Comment:
maybe we also need a `set -e` if we're doing multiple lines, if the `cd` fails then the `rm` will still execute